### ISSUE TITLE
False positive unexpected-keyword-arg with scipy.stats 1.17

### ISSUE THREAD

**Issue by sam-s** [pylint-dev/pylint#10831]
### Bug description

[`scipy.stats.entropy`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.entropy.html) accepts `nan_policy` but pylint does not seem to know it:

```python
"https://github.com/pylint-dev/pylint/issues/10831"
import numpy as np
from scipy import stats as st
print(st.entropy([1,2,3,np.nan], nan_policy="omit"))
```

This is a regression with SciPy Version: 1.17.0 released January 11, 2026.
The warning does not appear with `scipy-1.16.3`.

### Command used

```shell
pylint pylint-10831.py
```

### Pylint output

```python
pylint-10831.py:4:6: E1123: Unexpected keyword argument 'nan_policy' in function call (unexpected-keyword-arg)
```

### Expected behavior

no diagnostics, the code works as expected and prints 
```
np.float64(1.0114042647073518)
```

### Pylint version

```shell
pylint 4.0.4
astroid 4.0.3
Python 3.14.2 (tags/v3.14.2:df79316, Dec  5 2025, 17:18:21) [MSC v.1944 64 bit (AMD64)]
```

### OS / Environment

windows

### Additional dependencies

```python
numpy==2.4.1
scipy==1.17.0
```
